make-release.sh

```diff
@@ -35,7 +35,7 @@ bump_version () {
   echo "${VERSION}" > VERSION
   git add VERSION
   COMMIT_MSG="[release] Bump to ${NEXT_VERSION} in ${BUMP_BRANCH}"
-  git commit -asm "${COMMIT_MSG}"
+  git commit -asm "${COMMIT_MSG}" || true
```

Comment:
It may be safer option if we check if there is anything to commit before executing a command, otherwise commit may fail due other reasons and it won't be visible.

We already have such check in our PR check https://github.com/devfile/devworkspace-operator/blob/main/.github/workflows/pr.yml#L69

Reply:
if you want improvement, then we should check if version is already the good one and not try to write it again :-)

Reply:
see next commit. better? I was being quick-fix-lazy but your sane counterarguments have swayed me to making a more intelligent fix :)